modules/virt-sharing-custom-vm-template.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+:_content-type: PROCEDURE
+[id="virt-sharing-custom-vm-template_{context}"]
+= Sharing a custom virtual machine template
+
+Share virtual machine templates to the public {product-title} namespace so that you minimize creating multiple templates in private namespaces.
+
+.Prerequisites
+
+* You must have cluster administrator privileges to share virtual machine templates to the public {product-title} namespace.
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtulization*.
+. In the *Templates* tab, click the kebab icon and select *Share Template*.
+. In the Share template to {product-title} namespace window, click *Share Template*.
+
+.Verification
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtulization*.
+. In the *Templates* tab, display the *Namespace* column.
+// How does the user display the column in the UI?
+. Verify that the template namespace is *openshift*.
```

Comment:
A better verification would be to filter by template type > Shared. (screenshot in the Google doc)

Reply:
![image](https://user-images.githubusercontent.com/31725355/176171610-1834b1b4-dd06-4a26-a311-c6bb0ab85c20.png)